### DISCUSSION TITLE
[Question] What is the best practice in translating a sentence containing a link?

### DISCUSSION THREAD

**Discussion by NevenLiang** [ngneat/transloco#531] [Q&A] [answered]
### Question

For example, I have to translate a sentence with a link "Click [here](abc.com) to learn more".

```
// HTML
Click <a href='abc.com'>here</> to learn more
  1                      2           3
```

If I separete the texts in HTML into three parts. They are "Click", "here", and "to learn more". Usually it works if the translation result keeps the correct meaning with the order in English. However, the translation in Japanese has a different order. If we keep the order in English, it is wrong in other languages.

The [result](https://translate.google.com/?sl=en&tl=ja&text=click%20here%20to%20learn%20more&op=translate) from Google Translate is 

```
詳細については、ここをクリックしてください 

詳細 = more infomation
ここ = here
```

Our goal is displaying "here" as a link and make sure the translations in other languages have a correct meaning. Do we have a better way to handle this?

**Comment by shaharkazaz** [comment, marked as answer]
@NevenLiang you can pass the translated string into an `innerHTML` binding.

**Reply by NevenLiang** [reply]
@shaharkazaz Yeah, it could be an approach. 

I am thinking if we can handle it more elegantly with transloco features.
https://stackoverflow.com/questions/2543936/rails-i18n-translating-text-with-links-inside

**Reply by shaharkazaz** [reply]
Currently we don't have any special syntax or something

**Reply by NevenLiang** [reply]
Got it. Thanks!
